doc/languages-frameworks/rust.section.md

```diff
@@ -939,3 +939,67 @@ Fenix also has examples with `buildRustPackage`,
 [crane](https://github.com/ipetkov/crane),
 [naersk](https://github.com/nix-community/naersk),
 and cross compilation in its [Examples](https://github.com/nix-community/fenix#examples) section.
+## Using `git bisect` on the Rust compiler {#using-git-bisect-on-the-rust-compiler}
```

Comment:
I think some markdown implementation want to have new lines between sections? Not sure if I recall this correctly

```suggestion

## Using `git bisect` on the Rust compiler {#using-git-bisect-on-the-rust-compiler}
```